_extras/guide.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+## Setup before the lesson
+The required python packages for this lesson often result in installation issues,
+so it is advisable to organize a pre-workshop setup session where learners can show their installation and get help with problems.
+
+Although installation on learners own devices have the advantage of lowering the threshold to continue with the material beyond the workshop, this lesson can also be taught on a cloud environment such as [Google colab](https://colab.research.google.com/) or [My Binder](https://github.com/carpentries/scaffolds/blob/master/instructions/workshop-coordination.md#my-binder). This can serve as a backup environment when local installation fails. Some cloud environments offer the possibility to run the code on a GPU, which significantly increases the runtime of deep learning code.
```

Comment:
```suggestion
Installations on learners' devices have the advantage of lowering the threshold to continue with the material beyond the workshop. Note though, that this lesson can also be taught on a cloud environment such as [Google colab](https://colab.research.google.com/) or [My Binder](https://github.com/carpentries/scaffolds/blob/master/instructions/workshop-coordination.md#my-binder). This can serve as a backup environment when local installations fail. Some cloud environments offer the possibility to run the code on a GPU, which significantly increases the runtime of deep learning code.
```